modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -150,7 +151,7 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo FlowReversedInfo = {
-        "Reversed",
+        "ReversedFlow",
```

Comment:
This is a breaking change that we need to make sure that none of our assets are using the old version and it needs to be in the breaking changes list of the internal wiki

Reply:
Would say:

The `RenderableFieldlinesSequence`s `Reversed` property was renamed to `ReversedFlow` to match the input parameter name